samples/suit/recovery/README.rst

```diff
@@ -10,41 +10,40 @@ SUIT recovery application
    :depth: 2
 
 The SUIT recovery application is a minimal application that allows recovering the device firmware if the original firmware is damaged.
```

Comment:
Maybe would be worth to mention that it can recover only app and radio core if they are damaged.

Reply:
Ok, added this information